src/app/components/Timestamp/__snapshots__/index.test.jsx.snap

```diff
@@ -22,10 +92,13 @@ exports[`Timestamp should render Timestamp with datetime attribute and text 1`]
   }
 }
 
-<time
+<span
   className="c0"
-  dateTime="1530947227000"
 >
-  7 July 2018
-</time>
+  <time
```

Comment:
In this case, we'd want to render only the `<time>` tag, rather than having it wrapped in a `<span>`.